src/mgr/DaemonServer.cc

```diff
@@ -123,7 +123,16 @@ int DaemonServer::init(uint64_t gid, entity_addrvec_t client_addrs)
 			   "mgr",
 			   Messenger::get_random_nonce());
   msgr->set_default_policy(Messenger::Policy::stateless_server(0));
-
+  // throttle policy
+  msgr->set_policy(entity_name_t::TYPE_OSD,
+                   Messenger::Policy::stateless_server(
+                     CEPH_FEATURE_SERVER_LUMINOUS));
+  msgr->set_policy(entity_name_t::TYPE_MON,
+                   Messenger::Policy::lossy_client(CEPH_FEATURE_UID |
+                                                   CEPH_FEATURE_PGID64));
+  msgr->set_policy(entity_name_t::TYPE_MDS,
+                   Messenger::Policy::lossless_peer(CEPH_FEATURE_UID));
```

Comment:
```suggestion
  msgr->set_policy(entity_name_t::TYPE_MDS,
                    Messenger::Policy::stateless_server(
                     CEPH_FEATURE_SERVER_LUMINOUS));
```

I believe this is correct. For this messenger instance, the mgr should only be receiving perf metrics from the MDS.

Reply:
You're right